versions/3.0.3.md

```diff
@@ -2093,7 +2093,7 @@ solely by the existence of a relationship.
 ##### OperationRef Examples
 
 As references to `operationId` MAY NOT be possible (the `operationId` is an optional 
-value), references MAY also be made through a relative `operationRef`:
+value in an [Operation Object](#operationObject)), references MAY also be made through a relative `operationRef`:
```

Comment:
```suggestion
field in an [Operation Object](#operationObject)), references MAY also be made through a relative `operationRef`:
```

Reply:
Can't we just make `operationId` into a link to `#operationId` without adding the extra text?

Reply:
We could do, but as `operationId` exists in both objects, I think the extra text is worthwhile for those reading the sentence without hovering or clicking on the link.

Reply:
Duh, missed that 🤦‍♂️